lib/patch-series.ts

```diff
@@ -920,7 +906,7 @@ export class PatchSeries {
                                      this.project.branchName],
                                     { workDir: this.project.workDir });
         const args = [
-            "format-patch", "--thread", "--stdout", "--signature=gitgitgadget",
+            "format-patch", "--thread", "--stdout", `--signature=${this.config.repo.owner}`,
```

Comment:
If `config.repo.owner` is bad, it would probably break a number of things.  We could add a validation check after `--config` is loaded is `misc-helper`.

Reply:
> We could add a validation check after `--config` is loaded is `misc-helper`.

Sounds good!